PWGCF/DataModel/DptDptFiltered.h

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+#ifndef O2_ANALYSIS_DPTDPTFILTERED_H
+#define O2_ANALYSIS_DPTDPTFILTERED_H
+
+#include "Framework/ASoA.h"
+#include "Framework/AnalysisDataModel.h"
+
+namespace o2
+{
+namespace aod
+{
+/* we have to change from int to bool when bool columns work properly */
+namespace dptdptfilter
+{
+DECLARE_SOA_COLUMN(EventAccepted, eventaccepted, uint8_t); //! If the collision/event has been accepted or not
+DECLARE_SOA_COLUMN(EventCentMult, centmult, float);        //! The centrality/multiplicity pecentile
+} // namespace dptdptfilter
+DECLARE_SOA_TABLE(AcceptedEvents, "AOD", "ACCEPTEDEVENTS", //! Accepted reconstructed collisions/events filtered table
```

Comment:
You may want to call it Collision instead of Event to follow the usual nomenclature.
Also this name has to be unique ACCEPTEDEVENTS in O2, so let's call it CFDPTCOLLISONS or similar

Reply:
Good point!
I will rename them